buildSrc/src/test/kotlin/com/example/model/Version.kt

```diff
@@ -78,6 +83,8 @@ public data class Version(
     public class Id() {
         public val serialNumber: Number = 12112.0
 
+        @Suppress("CyclomaticComplexMethod", "LongMethod", "ThrowsCount",
```

Comment:
don't you think it is too much to apply the detekt rules on the generated code  ?